### PR TITLE
Registering the Bangalore Azure community

### PR DESCRIPTION
**Pull request by Keshav1308**
Registering the Bangalore Azure community

### CODE REVIEW

File: 2023/AzureBengaluru/data.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "ValidMaintainers": {
+    "Description": "GitHub users who maintain this folder. PR for changing this folder will only be accepted from these users.",
+    "GitHubHandles": [
+      "",
+      "",
+      ""
```
Comment (noopman) [inline 2023/AzureBengaluru/data.json:7]: We need the GitHub ID's of those users who are allowed to edit your folder! For example yours @Keshav1308

File: 2023/AzureBengaluru/data.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    ]
+  },
+  "CommunityInfo": {
+    "Name": "Bengaluru Azure Community",
+    "Country": "India",
+    "Url": "https://bengaluru.globalazure.in",
+    "SessionizeApiId": "10862",
```
Comment (noopman) [inline 2023/AzureBengaluru/data.json:14]: Are you sure this SessionizeID is accurate?

File: 2023/AzureBengaluru/data.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    "Description_SessionizeApiId": "If you have configured your Sessionize correctly then browsing to this url (replacing {Your sessionize event Id} with your event sessionize id) will return valid json: https://sessionize.com/api/v2/10862/view/Sessions",
+    "SessionizeEventName": "{Your sessionize event name (or complete URL to the public Sessionize C4P, i.e. https://sessionize.com/global-azure or just global-azure)}",
+    "Logo": {
+      "Description": "Community logo is a .png file 500X500px named AzureBengaluru.png.",
+      "File": "AzureBengaluru.png"
```
Comment (noopman) [inline 2023/AzureBengaluru/data.json:19]: This file is not present in your submission. Instead you have a file _template.png in the folder, and that is just a placeholder.

**Review by noopman — CHANGES_REQUESTED** [review]
Hello @Keshav1308 ! Thank you for your interest in Global Azure 2023 and for claiming your community pin on the map. As there are a few items missing from your submission, please review our [readme](https://github.com/globalazure/communities#global-azure-communities) to learn how to complete your registration.

We need a logo file, an index.md file and the data in your data.json needs a few fixes!

Thanks in advance!

**Review by noopman — CHANGES_REQUESTED** [review]
This is looking very good indeed! Thanks for the progress on this @Keshav1308 and @samikroy.

1. Can you verify the "SeeionizeApiId" please. I'm not sure it looks right. When I check it the result is 404.
2. I need an index.md file in the folder for you! Please look in other folders next to yours for inspiration what other communities did!

**Comment by Keshav1308** [comment]
Thanks, Magnus, for looking into this.  attaching the index file. Use this latest index file please 🙂 .

This Sessionize ID is working for us https://sessionize.com/app/organizer/event/10862

[cid:bc859433-e6e1-4678-a8e0-37be573caadf]

________________________________

**Comment by noopman** [comment]
> Thanks, Magnus, for looking into this. attaching the index file. This Sessionize ID is working for us https://sessionize.com/app/organizer/event/10862 [cid:bc859433-e6e1-4678-a8e0-37be573caadf]

@Keshav1308 this is still confounding me. Here is an example Sessionize ID that works: w84xg5cq. It can be verified by browsing to the API endpoint of the event like this: [https://sessionize.com/api/v2/w84xg5cq/view/Sessions](https://sessionize.com/api/v2/w84xg5cq/view/Sessions) The endpoint is present and shows that there are currently no published sessions at this event.

When I do the same with your ID this is the result: [https://sessionize.com/api/v2/10862/view/Sessions](https://sessionize.com/api/v2/10862/view/Sessions) 404!

I think you are using the event ID but not the API ID. Please look into your sessionize event and see if you can find the event's unique API ID!

**Comment by noopman** [comment]
You also need, please, to add an index.md file to your folder in this PR!

**Comment by Keshav1308** [comment]
Adding Samik Roy.

Hi @***@***.******@***.***> ,

We need the event's unique API ID!

Regards,
Keshav Jain.
________________________________

**Comment by noopman** [comment]
Great with the index.md file. It looks very good. Now we only need to ensure you have the right Sessionize API ID in the file!

**Comment by samikroy** [comment]
> Great with the index.md file. It looks very good. Now we only need to ensure you have the right Sessionize API ID in the file!

@noopman  - Thank you for sharing this.
Could you please have a look if this one you are looing for
https://sessionize.com/api/v2/o6kupum9/view/Sessions
CC- @Keshav1308

**Review by noopman — APPROVED** [review]
That sessionizeid looks right @Keshav1308 and @samikroy ! I went ahead and added it to your data.json file for you, and am now approving your PR! Thanks for joining Global Azure!

**Pull request merged**